### PR TITLE
Use console.php for entering/exiting maintenance mode.

### PR DESCRIPTION
**Pull request by orhtej2**
## Problem

- Backup script does not enter maintenance mode as per #111 

## Solution

- `bin/console` is a shell script, `bin/console.php` is a thing that needs to be run with PHP interpreter.
- if backup fails maintenance mode is not disabled - use `trap` to deal with it.

## PR Status

- [x] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by orhtej2** [comment]
!testme

File: scripts/backup [hunk 1]
```diff
@@ -61,8 +63,6 @@ ynh_print_info --message="Backing up the MySQL database..."
 
 ynh_mysql_dump_db --database="$db_name" > db.sql
 
-(cd $install_dir && ynh_exec_as "$app" php$phpversion bin/console maintenance 0)
```
Comment (MrPetovan) [inline scripts/backup:64]: This still needs to be done in case of a successful run.

Comment (orhtej2) [reply scripts/backup:64]: It's now part of `trap` hence will happen no matter if `backup` succeeds or fails.

**Comment by orhtej2** [comment]
!testme

**Comment by orhtej2** [comment]
Seems to be working fine https://paste.yunohost.org/raw/hafepevile

**Review by MrPetovan — APPROVED** [review]

**Comment by orhtej2** [comment]
!testme

**Comment by orhtej2** [comment]
> Seems to be working fine https://paste.yunohost.org/raw/hafepevile

It was backing up OK but restoring in maintenance mode hence `restore` test was failing.

**Comment by orhtej2** [comment]
(╯°□°)╯︵ ┻━┻

**Comment by orhtej2** [comment]
!testme

**Review by MrPetovan — APPROVED** [review]

**Pull request merged**